api/app/reviews/api.py

```diff
@@ -79,7 +79,26 @@
     'response_id': fields.Integer,
     'reviewer_user_id': fields.Integer,
     'scores': fields.List(fields.Nested(review_scores_fields), attribute='review_scores'),
-    'language': fields.String
+    'language': fields.String,
+    'is_submitted': fields.Boolean,
+    'submitted_timestamp': fields.DateTime(dt_format='iso8601')
+}
+
+extended_scores_fields = {
+    'review_question_id': fields.Integer, 
+    'headline': fields.String, 
+    'description': fields.String, 

```

Comment:
Are `headline` and `description` needed considering you can get them from the review form? (not too concerned about keeping it like this if it makes it easier for you though)